### ISSUE TITLE
CLI executable

### ISSUE THREAD

**Issue by ilyakaznacheev** [go-pg/migrations#70]
Does the library have a ready-to-use executable somewhere?
To just download&run like
```bash
# download and install the executable
go get github.com/go-pg/migrations/cmd/pg-migrations
# run migrations as an app
pg-migrations init
pg-migrations up 2
```

**Comment by vmihailenco** [comment]
No, because this lib needs database config and requiring everyone to use same CLI, config format and single database does not sound reasonable. Just copy the bits you need from https://github.com/go-pg/migrations/blob/master/example/main.go and use CLI/config you prefer.

**Comment by ilyakaznacheev** [comment]
So why don't implement a simple CLI wrapper to allow new users just install it and use like [golang-migrate](https://github.com/golang-migrate/migrate/tree/master/cmd/migrate) does?

I don't mean package building or installing with some package manager like brew or apt, but just a standalone CLI app, that will allow to run it without any code manipulation.

**Comment by movergan** [comment]
I have to agree, having a binary would be awesome.
